workflow-service/src/main/java/com/redhat/parodos/workflow/execution/service/WorkFlowService.java

```diff
@@ -39,11 +39,17 @@ public interface WorkFlowService {
 
 	WorkReport execute(WorkFlowRequestDTO workFlowRequestDTO);
 
+	WorkReport restart(UUID workFlowExecutionId);
+
 	WorkFlowExecution getWorkFlowById(UUID workFlowExecutionId);
 
 	WorkFlowExecution saveWorkFlow(UUID projectId, UUID userId, WorkFlowDefinition workFlowDefinition,
 			WorkStatus workStatus, WorkFlowExecution mainWorkFlowExecution, String arguments);
 
+	WorkFlowExecution saveWorkFlow(UUID projectId, UUID userId, WorkFlowDefinition workFlowDefinition,
```

Comment:
do you think naming this one as `saveRestartedWorkFlow` makes more sense?

Reply:
It adds a bit more context as why this method should be used for, I'll rename it